HowTos/SAML_Integration_Okta_IdP.rst

```diff
@@ -132,9 +132,14 @@ Step 3. Retrieve Okta IdP metadata
 
 #. After the application is created in Okta, go to the `Sign On` tab for the application.
 
-#. Copy the URL from the *Identity Provider metadata* link. This value will be used to configure the Aviatrix SP Endpoint.
+#. Scoll down to *SAML Signing Certificates* section, notice SHA-2 certificate is Active, click on Actions -> View IdP metadata
```

Comment:
- Change "Scoll" to "Scroll" 
- Remove italics for "SAML Signing Certificates" 
- "notice that the SHA-2 certificate is Active" 
- "click Actions > View IdP metadata"